base/xx-apt

```diff
@@ -45,6 +45,50 @@ else
   arg0="apt"
 fi
 
+fixubuntusources() {
+  # fix ubuntu sources
+  if grep "# generated by xx-apt" /etc/apt/sources.list >/dev/null 2>/dev/null; then
+    return
+  fi
+
+  group1="amd64,i386"
+  group2="arm64,armhf,ppc64el,s390x"
+
+  nativearch="$(TARGETPLATFORM="" TARGETPAIR="" TARGETARCH="" TARGETOS="" xx-info arch)"
+
+  while IFS="" read -r p || [ -n "$p" ]; do
+
+    if echo "$p" | grep -E "^(deb|deb-src) "; then
+      case "${nativearch}" in
+        amd64 | 386)
```

Comment:
This fixes all active rules to the current arch group and adds a duplicate rule for the additional group.

Alternative would be to set current rules only to the current native arch and add detect the ubuntu distro and add only the known rules for that arch, each time fixed to individual arch.

Maybe the second one is better. wdyt? @crazy-max

Reply:
> Alternative would be to set current rules only to the current native arch and add detect the ubuntu distro and add only the known rules for that arch, each time fixed to individual arch.

Yes, the second solution with native arch might be better in the long run.

Reply:
Let's merge this one. I'll make a new PR and we can see if it actually improves.